src/server/birds/helpers/fetch/find-tracking-locations.js

```diff
@@ -0,0 +1,20 @@
+// import { trackingGet } from '~/src/server/birds/helpers/fetch/tracking-fetcher'
+
+async function findTrackingLocations(birdId, trackingId) {
+  //   return await trackingGet(birdId, trackingId, '/locations')
+  // TODO
```

Comment:
Will disuss this. As the backend can not access the frontend's S3 at the moment.